pkg/monitoring/client/prometheus/prometheus.go

```diff
@@ -54,9 +59,35 @@ var (
 		},
 		[]string{"code", "method", "host"},
 	)
+
+	resourceRequestsTotal = prometheus.NewCounterVec(prometheus.CounterOpts{
+		Name: "rest_client_operation_and_resource_requests_total",
```

Comment:
@davidvossel Please add kubevirt_ prefix

Reply:
Isn't that something we can do at the exporter level so we don't have to think about it every time?
You can tell the prometheus lib to always prefix metrics with e.g. `kubevirt`.

Reply:
> Please add kubevirt_ prefix

i'll add the kubevirt prefix. 

> Isn't that something we can do at the exporter level so we don't have to think about it every time?
You can tell the prometheus lib to always prefix metrics with e.g. kubevirt.

not all our metrics should have the prefix. some of the metrics are expanding existing k8s metrics, like `rest_client_request_latency_seconds` for example

Reply:
i added the kubevirt prefix